modules/primer-avatars/lib/avatar-stack.scss

```diff
@@ -43,3 +43,134 @@
     }
   }
 }
+
+// Refactored, new avatar stack:
+
+.AvatarStack {
+  position: relative;
+  min-width: 26px;
+  height: 20px;
+
+  &.AvatarStack--2 {
+    min-width: 36px;
+  }
+
+  &.AvatarStack--3 {
+    min-width: 46px;
+  }
+}
+
+.AvatarStack-body {
+  position: absolute;
+  display: flex;
+  background: $bg-white;
+
+  .avatar {
+    position: relative;
+    z-index: 2;
+    display: flex;
+    width: 20px;
```

Comment:
Use `$spacer-4` for the width and height instead of static values.

Reply:
`$spacer-4` is 24px. these need to be 20px to work in all the various views across the app.

Reply:
Sorry I derped! Please leave as is.